content/en/users/datatransfer/WebFTS/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "WebFTS"
+linkTitle: "WebFTS"
+type: docs
+description: "Documentation related to EGI Data Transfer Web Interface"
+weight: 30
+---
+
+## Overview 
+
+The WebFTS interface offers end users a way to graphically execute and monitor file 
+transfer between storages using different protocols. You can check CERN WebFTS 
+[documentation](https://fts3-docs.web.cern.ch/fts3-docs/docs/webfts/userguide.html) and 
+[FAQ](https://fts3-docs.web.cern.ch/fts3-docs/docs/webfts/faq.html) for more details.
+
+
+### Credential Delegation
+
+In order to move files between storage endpoints, named also Storage Elements (SE), 
+the authentication is done using an X509 certificate which needs to be installed on the browser. 
+The Integration with EGI Check-in is under development and will be available at a latter date in the production instance.
```

Comment:
Please try to use the same line length everywhere, this one is longer than the others :)